loom/script/native/core/system/lmJSON.cpp

```diff
@@ -645,5 +723,5 @@ static int registerSystemJSON(lua_State *L)
 
 void installSystemJSON()
 {
-    NativeInterface::registerNativeType<JSON>(registerSystemJSON);
+    NativeInterface::registerManagedNativeType<JSON>(registerSystemJSON);
```

Comment:
Wouldn't backing out on the script side mean removing the script-side LS-only methods? Removing the entire point of the pull? :P

It's probable I'm not entirely understanding what a manage native type entails.